ted2zim/scraper.py

```diff
@@ -2,42 +2,42 @@
 # -*- coding: utf-8 -*-
 # vim: ai ts=4 sts=4 et sw=4 nu
 
-import dateutil.parser
+import concurrent.futures
+import datetime
 import json
-import time
+import locale
 import pathlib
 import shutil
 import tempfile
-import datetime
+import time
 import urllib.parse
-import concurrent.futures
+from gettext import gettext as _
```

Comment:
you should import it from `zimscraperlib.i18n`

Reply:
Okay. Got it.